tools/acrn-manager/acrnctl.c

```diff
@@ -193,6 +193,21 @@ static int acrnctl_do_add(int argc, char *argv[])
 			printf("Too many optional args: %s\n", args);
 			return -1;
 		}
+
+		/*
+		 * If there's "-C" parameter in acrnctl add command
+		 * check if the SoS support runC container at first, then
+		 * strip "-C" and set the flag.
+		*/
+		if (strncmp(argv[i], "-C", 2) == 0) {
+			if (access("/sbin/runc", F_OK) != 0) {
+				printf("runC command not supproted\n");
```

Comment:
How about: `Your system does not appear to have runc installed, please install it first. Exiting.`

Reply:
The function only works on SOS which is above specific version,  so even users install the runc, it will not work, so if there's  no runc binary in /sbin, we suggest user don't use "-C" parameter.

Reply:
We need to be more agnostic to the SOS version. Other OSs (Linux distros) could be used instead of Clear Linux which we use as the reference today. Hard-coding things the way it's done is not good practice. This, for example, will break things with Ubuntu for no other reason that we're checking the presence of a tool in the wrong location (Ubuntu puts it under `/usr/sbin` and not `/sbin`), but the feature would otherwise be fully functional.

Reply:
OK, I will change the logical in the next patch. 
Thanks